FhirToDataLake/docs/Deployment.md

```diff
@@ -90,9 +90,11 @@ Running the PowerShell script that creates following artifacts:
 To run the PowerShell Script, perform the following steps:
 
 1. Ensure that you have the latest version of the PowerShell.
-1. Clone this [GitHub](https://github.com/microsoft/FHIR-Analytics-Pipelines) repo to your local machine.
-1. Open the Azure CLI and browse to the scripts folder.
+1. Open the Azure CLI.
 1. Sign in to your Azure account using `Connect-AzAccount -SubscriptionId 'yyyy-yyyy-yyyy-yyyy'`. Use your subscription ID.
+1. Run `Install-Module -Name Az.Synapse`. This is to ensure that you have the latest version of the Az.Synapse sub-module as it is not updated by default. 
+1. Clone this [GitHub](https://github.com/microsoft/FHIR-Analytics-Pipelines) repo inside your Azure CLI folder.
+1. Browse to the scripts folder under this path (..\FHIR-Analytics-Pipelines-main\FhirToDataLake\scripts).
 1. Run `./Set-SynapseEnvironment.ps1 -SynapseWorkspaceName "{Name of your Synapse workspace instance}" -StorageName "{Name of your storage account where Parquet files are written}"`. Refer to the complete syntax below.
```

Comment:
Maybe we can directly use Powershell Az modules rather than Azure CLI? because ``Set-SynapseEnvironment.ps1`` uses Powershell Az modules inside.

From my test on new clean environment, the steps can be 




1. Clone this [FHIR-Analytics-Pipelines](https://github.com/microsoft/FHIR-Analytics-Pipelines) repo to your local machine.
1. Open the PowerShell console, ensure that you have the latest version of the PowerShell.
1. Install [Az](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-7.1.0) or separated [Az.Synapse]() if they don't exist.

    ```Powershell
    Install-Module -Name Az
    Install-Module -Name Az.Synapse
    ```
1. Sign in to your Azure account using

    ```Powershell
    Connect-AzAccount -SubscriptionId 'yyyy-yyyy-yyyy-yyyy'
    ``` 
1. Browse to the scripts folder under this path ```...\FhirToDataLake\scripts```.
1. Run the ``Set-SynapseEnvironment.ps1`` script, refer to the complete syntax below for details of parameters.

    ```Powershell
    ./Set-SynapseEnvironment.ps1 -SynapseWorkspaceName "{Name of your Synapse workspace instance}" -StorageName "{Name of your storage account where Parquet files are written}".
    ```


Add @ranvijaykumar  for notice.

Reply:
Hi @MustAl-Du, is this PR still active? if so, can you help to update the steps section then we can merge and fix the document :). otherwise I may need to update it in another PR, cause there will be some problems when users create synapse environment according to the current documents

Reply:
Thanks @QuanWanxx - I will issue another PR with the steps described in your note above.